codemodel/.resources/all-in-one/json/code-model.json

```diff
@@ -1710,6 +1731,13 @@
             "$ref": "#/definitions/DateSchema"
           }
         },
+        "times": {
+          "description": "a time",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiVersionimeSchema"
```

Comment:
Something weird happened with the definition name here: `ApiVerisonimeSchema`

Reply:
O_O